pages/linux/cryptsetup.md

```diff
@@ -4,8 +4,8 @@
 
 - Format a drive (or a file) to make it a LUKS volume:
 
-`cryptsetup luksFormat {{path_to_luks_volume_to_be}}`
+`cryptsetup luksFormat {{path/to/luks_volume_to_be}}`
```

Comment:
If I understand this example correctly, perhaps "path/to/new_luks_volume" would be more intuitive. But I may be misinterpreting the meaning of the token; let me know what you think.